src/EnergyPlus/ElectricPowerServiceManager.hh

```diff
@@ -833,6 +833,9 @@ void createFacilityElectricPowerServiceObject(const EnergyPlusData &state);
 
 Real64 checkUserEfficiencyInput(EnergyPlusData &state, Real64 userInputValue, std::string whichType, std::string deviceName, bool &errorsFound);
 
+void checkChargeDischargeVoltageCurves(
+    EnergyPlusData &state, std::string const nameBatt, Real64 const E0c, Real64 const E0d, int const chargeIndex, int const dischargeIndex);
+
```

Comment:
std::string_view is desired. std::string arguments are referenced, std::string_view are not.

Reply:
I'll change that in the next commit.

Reply:
Arguments look good now.

Reply:
:+1: